projects/crypto-currency/destemidos/App/src/components/Feeds/PostCard.jsx

```diff
@@ -1,14 +1,27 @@
-import React from 'react';
-
+import React from 'react'
+import Grid from '@material-ui/core/Grid'
+import Avatar from '@material-ui/core/Avatar'
+import Typography from '@material-ui/core/Typography'
+import TwitterIcon from '@material-ui/icons/Twitter'
+import { ContainerRoot, PaperDiv,TypographyDiv } from './Style.js'
 
 const PostCard = ({body, social_score, retweets, likes}) => {
```

Comment:
Poderia colocar espaço no começo e no final das chaves? 
```suggestion
const PostCard = ({ body, social_score, retweets, likes }) => {
```

Reply:
alterado